velocitas_lib/test/test_velocitas_lib.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2023 Robert Bosch GmbH
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import os
+import sys
+
+from velocitas_lib import (
+    get_app_manifest,
+    get_cache_data,
+    get_script_path,
+    get_workspace_dir,
+    json_obj_to_flat_map,
+    replace_variables,
+    require_env,
+)
+
+cache_data_mock = {"testPropA": "testValueA", "testPropB": "testValueB"}
+app_manifest = {"vehicleModel": {"src": "test"}}
+
+os.environ["TEST"] = "test"
+os.environ["VELOCITAS_WORKSPACE_DIR"] = "/test/vehicle-app-workspace"
+os.environ["VELOCITAS_APP_MANIFEST"] = json.dumps(app_manifest)
+os.environ["VELOCITAS_CACHE_DATA"] = json.dumps(cache_data_mock)
+
+
+def test_require_env():
+    assert require_env("TEST") == "test"
+
```

Comment:
There should be also a bad case test:
(Naming follows the pattern <triggered_action__condition__expected_behaviour>)
```suggestion


def test_require_env__env_var_not_set__raises_ValueError():
    assertRaises(ValueError, require_env, "SOME_MOST_PROBABLY_NEVER_SET_ENVIRONMENT_VARIABLE_NAME")
```